mockup/patterns/relateditems/pattern.js

```diff
@@ -350,6 +353,7 @@ define([
       self.treeQuery = new utils.QueryHelper(
         $.extend(true, {}, self.options, {
           pattern: self,
+          vocabularyUrl: self.options.treeOptions.vocabularyUrl,
```

Comment:
So I'm still not convinced this will cause other problems. I want it to fallback onto the vocabularyUrl for the related items pattern.

I think there are cases doing it this way where vocabularyUrl will end up being null(since you default treeOptions.vocabularyUrl to null). Can you remove that default value?